tests/vmi_multus_test.go

```diff
@@ -375,32 +375,50 @@ var _ = Describe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge CNI plugin interface and custom MAC address.", func() {
-			interfaces := []v1.Interface{linuxBridgeInterface}
-			networks := []v1.Network{linuxBridgeNetwork}
-			linuxBridgeIfIdx := 0
 			customMacAddress := "50:00:00:00:90:0d"
 
 			It("[test_id:676]should configure valid custom MAC address on Linux bridge CNI interface.", func() {
 				By("Creating a VM with Linux bridge CNI network interface and default MAC address.")
-				vmiTwo := createVMIOnNode(interfaces, networks)
-				tests.WaitUntilVMIReady(vmiTwo, console.LoginToAlpine)
+				vmiTwo := libvmi.NewFedora(
```

Comment:
nit: consider extracting the vmi object creation  to a helper that will receive networks and interfaces and return the vmi object. pretty much the same process is repeated here and on line 395, and also it is a long list of `libvmi.WithSomething`, and perhaps could be abstracted. I don't think it is that important, because it only repeated twice, but perhaps it will be useful in the future.

Reply:
Let's follow the rule of three and keep it like this for now.